src/video_core/command_classes/codecs/codec.cpp

```diff
@@ -99,12 +142,32 @@ void Codec::Decode() {
     packet.data = frame_data.data();
     packet.size = static_cast<s32>(frame_data.size());
 
-    avcodec_send_packet(av_codec_ctx, &packet);
+    int ret = avcodec_send_packet(av_codec_ctx, &packet);
```

Comment:
```suggestion
    const int ret = avcodec_send_packet(av_codec_ctx, &packet);
```

And make the other assignment use its own local variable, since the results aren't related to one another.

Reply:
Fixed the 3 places with a separate commit